fe/fe-core/src/main/java/org/apache/doris/catalog/Env.java

```diff
@@ -289,6 +291,9 @@ public class Env {
     public static final String CLIENT_NODE_HOST_KEY = "CLIENT_NODE_HOST";
     public static final String CLIENT_NODE_PORT_KEY = "CLIENT_NODE_PORT";
 
+    private static final String VERSION_FILE = "/VERSION";
```

Comment:
I think we could add a version_history folder in meta dir. And we could create multiple version files. like  1_2_3_commitid_version, 1_2_4_commitid_version,1_2_5_commitid_version, 1_2_6_commitid_version, 2_0_0_commitid_version
 
Then we could get the fe version history and commit id history.